lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -115,6 +115,10 @@ func websocketEchoHandler(w http.ResponseWriter, req *http.Request) {
 		if err != nil {
 			break
 		}
+		err = conn.Close()
```

Comment:
nit: I prefer one liner

```
if err := conn.Close(); err != nil {
    break
}
```

Reply:
I'd rather leave it as is, if you don't mind, since it matches the convention from the lines above.

But in general, I agree with your suggestion. I just don't want to change the rest of the function to match.

Reply:
Yes, it's a "nit", so feel free to ignore it ^^